keyboards/maartenwut/plain60/keymaps/default/keymap.c

```diff
@@ -5,13 +5,21 @@
 // Layer names don't all need to be of the same length, obviously, and you can also skip them
 // entirely and just use numbers.
 #define _MA 0
+#define _FN 0
```

Comment:
```suggestion
#define _FN 1
```

or the whole enum thing maybe?

Reply:
good catch